README_zh-hans.md

```
@@ -1,28 +1,68 @@
 **鉴于原作者已经归档，本仓库不会创建类似渠道。如遇到任何问题请于 [Github Issue](https://github.com/LenovoLegionToolkit-Team/LenovoLegionToolkit/issues) 进行反馈，谢谢。**
 
-<img src="assets/Main_DarkMode.png" width="700" />
+# 本地化
+本分支已接入 Crowdin：[LenovoLegionToolkit](https://crowdin.com/project/lenovolegiontoolkit-unofficial)。
+
+欢迎并感谢大家参与本地化翻译！
+
+# 截图
+
+### 主窗口
+<img src="assets/Main_DarkMode.png" width="49%" /> <img src="assets/Main_LightMode.png" width="49%" />
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)